internal/core/unittest/test_minio_chunk_manager.cpp

```diff
@@ -108,15 +110,16 @@ TEST_F(MinioChunkManagerTest, ObjectExist) {
 }
 
 TEST_F(MinioChunkManagerTest, WritePositive) {
-    string testBucketName = "test-write";
+    string testBucketName = configs_.bucket_name;
     chunk_manager_->SetBucketName(testBucketName);
     EXPECT_EQ(chunk_manager_->GetBucketName(), testBucketName);
 
-    if (!chunk_manager_->BucketExists(testBucketName)) {
-        chunk_manager_->CreateBucket(testBucketName);
-    }
+    // if (!chunk_manager_->BucketExists(testBucketName)) {
+    //     chunk_manager_->CreateBucket(testBucketName);
+    // }
+    auto has_bucket = chunk_manager_->BucketExists(testBucketName);
     uint8_t data[5] = {0x17, 0x32, 0x45, 0x34, 0x23};
-    string path = "1/3/5";
+    string path = "1";
```

Comment:
This path seems to have nothing to do with pr, why should it be modified？

Reply:
can ignore this test, both is ok.